src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollectionUtils.java

```diff
@@ -561,8 +563,51 @@ public static ReadCountCollection removeColumnsWithExtremeMedianCounts(final Rea
         }
     }
 
-    private static long countZeroes(final double[] data) {
-        return DoubleStream.of(data).filter(d -> d == 0.0).count();
+    /**
+     * Remove columns with NaNs and infinities
+     *
+     * @param readCounts
```

Comment:
Actually, I would just copy + paste the javadoc from the other filtering methods in this class and modify accordingly.  ("`It might be a reference to the input read-counts if there is no column to be dropped.`")